ecosystem/typescript/sdk/package.json

```diff
@@ -51,7 +51,9 @@
   },
   "dependencies": {
     "axios": "^0.26.1",
+    "bip39-light": "^1.0.7",
```

Comment:
https://github.com/paulmillr/scure-bip39 might be a better fit. Runs in the browser without issue

Reply:
Used in [js-ethereum-cryptography](https://github.com/ethereum/js-ethereum-cryptography/blob/master/package.json)